main.go

```diff
@@ -80,7 +80,7 @@ func main() {
 	flag.BoolVar(&settings.ExchangePurgeCredentials, "exchangepurgecredentials", false, "purges the stored exchange API credentials")
 	flag.BoolVar(&settings.EnableExchangeHTTPRateLimiter, "ratelimiter", true, "enables the rate limiter for HTTP requests")
 	flag.IntVar(&settings.MaxHTTPRequestJobsLimit, "requestjobslimit", int(request.DefaultMaxRequestJobs), "sets the max amount of jobs the HTTP request package stores")
-	flag.IntVar(&settings.RequestTimeoutRetryAttempts, "exchangehttptimeoutretryattempts", request.DefaultTimeoutRetryAttempts, "sets the amount of retry attempts after a HTTP request times out")
+	flag.IntVar(&settings.RequestMaxRetryAttempts, "exchangehttpmaxretryattempts", request.DefaultMaxRetryAttempts, "sets the amount of retry attempts after a retryable HTTP failure")
```

Comment:
as the requestor package and these new rate limit changes are used outside of exchanges the exchange prefix on the command (and the 3 below) can be dropped now 

What are everyone else's thoughts on this?>

Reply:
Can do 👍 I'd guess that we want to remove the mention of 'exchange' from the `settings` field, the flag name and the flag description?

Reply:
Noticed that there seems to be some overlap between these and 'global' HTTP Timeout / 
 User Agent / Proxy?

```
gctlog.Debugf(gctlog.Global, "- COMMON SETTINGS:")
	gctlog.Debugf(gctlog.Global, "\t Global HTTP timeout: %v", s.GlobalHTTPTimeout)
	gctlog.Debugf(gctlog.Global, "\t Global HTTP user agent: %v", s.GlobalHTTPUserAgent)
gctlog.Debugf(gctlog.Global, "\t Global HTTP proxy: %v", s.ExchangeHTTPProxy)
```

Reply:
The difference between them is that a) the exchange related ones aim to adjust the exchange requests user agent / proxy settings / timeout as these are all exchange config adjustable and the global settings are for all other subsystems (like the forex handler, SMS gateway etc).  In a different PR, it would be good to have one settings which adjusts both, so everything is unified but out of scope for this PR IMO